paket.dependencies

```diff
@@ -1,25 +1,41 @@
 redirects: on
-framework: auto-detect
+framework: net45, net452, netstandard2.0, netcoreapp2.1
```

Comment:
It is some kind of defect that it needs to be listed?

Do we want to reduce the count (like having only net45 or net452 and the .NET Core ones)?